src/nbl/video/CVulkanPhysicalDevice.cpp

```diff
@@ -895,6 +900,32 @@ std::unique_ptr<CVulkanPhysicalDevice> CVulkanPhysicalDevice::create(core::smart
         properties.limits.shaderFloat16 = vulkan12Features.shaderFloat16;
         if (!vulkan12Features.shaderInt8)
             return nullptr;
+
+        if (properties.limits.shaderFloat16) {
+            if (!vulkan12Properties.shaderSignedZeroInfNanPreserveFloat16)
+                return nullptr;
+            if (!vulkan12Properties.shaderDenormPreserveFloat16)
+                return nullptr;
+            if (!vulkan12Properties.shaderDenormFlushToZeroFloat16)
+                return nullptr;
+            if (!vulkan12Properties.shaderRoundingModeRTEFloat16)
+                return nullptr;
+            if (!vulkan12Properties.shaderRoundingModeRTZFloat16)
+                return nullptr;
+        }
+
+        if (properties.limits.shaderFloat64) {
+            if (!vulkan12Properties.shaderSignedZeroInfNanPreserveFloat64)
+                return nullptr;
+            if (!vulkan12Properties.shaderDenormPreserveFloat64)
+                return nullptr;
+            if (!vulkan12Properties.shaderDenormFlushToZeroFloat64)
+                return nullptr;
+            if (!vulkan12Properties.shaderRoundingModeRTEFloat64)
+                return nullptr;
+            if (!vulkan12Properties.shaderRoundingModeRTZFloat64)
+                return nullptr;
```

Comment:
Denorm and RTZ are not all true on the RTX 3060
![image](https://github.com/user-attachments/assets/2d6c814d-40cc-4840-b628-2e458f8b85f0)